src/components/PrimaryNav/PrimaryNav.js

```diff
@@ -94,6 +98,11 @@ const PrimaryNav = () => {
             })}
           >
             <Link className="p-list__link" to={navItem.path}>
+              <img
+                className="p-list__icon"
+                src={`https://assets.ubuntu.com/v1/${navItem.icon}`}
```

Comment:
We'll need to make sure that this can work offline if it's going to be part of Juju so we can't access the assets from the asset server.

Reply:
Issue raised here: https://github.com/canonical-web-and-design/jaas-dashboard/issues/101